Source/processcontainers/ProcessContainer.h

```diff
@@ -23,75 +23,62 @@
 
 namespace WPEFramework {
 namespace ProcessContainers {
-    
     using IStringIterator = Core::IteratorType<std::vector<string>, const string>;
-    using IConstStringIterator = Core::IteratorType<const std::vector<string>, const string, std::vector<string>::const_iterator>;
-
-    class INetworkInterfaceIterator : public Core::IIterator 
-    {
-    public:
-        virtual ~INetworkInterfaceIterator() = default;
```

Comment:
To keep it consistent with our codebase (I know it is not technically needed as d'tor already in base class) please keep the destructors (so also everywhere below) (and according to new rule here you would use override instead of virtual, but that is not yet everywhere in our code base)